packages/block-serialization-spec-parser/index.js

```diff
@@ -1,2 +0,0 @@
-export { parse } from './parser';
```

Comment:
Do we need the changes in the spec and default parsers? I don't understand why they're necessary. I've undone them, and could `npm install`, `npm run test`, and `npm run dev` without any problem (also tested to load and save an existing post).

Reply:
Yes, I had the same issue as it’s no longer bundled into Gutenberg. There is another parser implementation (more performant one) that is bundled by default. You would have to change it in both PHP and JS files related to parser in wp-blocks. It explodes with umd.